### ISSUE TITLE
PHP Warning with WP trunk

### ISSUE THREAD

**Issue by jeherve** [Automattic/vaultpress#94]
The following changeset was recently committed to `trunk`:
https://core.trac.wordpress.org/changeset/48204

This is causing warnings in VaultPress:

```
Warning: Declaration of VaultPress_IXR_SSL_Client::query() should be compatible with IXR_Client::query(...$args) in /wp-content/plugins/vaultpress/class.vaultpress-ixr-ssl-client.php on line 25
```

**Comment by gmazzap** [comment]
I still see this warning on a VIP Go environment. Is this not merged yet?

**Comment by jeherve** [comment]
@gmazzap It is. It was shipped with VaultPress 2.1.4. Could you double check to see what version of VaultPress is in use on VIP Go?

**Comment by gmazzap** [comment]
Looks like 2.1.1 https://github.com/Automattic/vip-go-mu-plugins/blob/master/vaultpress/vaultpress.php#L6
